cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -53,6 +56,30 @@ func cleanupContainerDiskDirectory(ephemeralDiskDir string) {
 	}
 }
 
+func dumpLogFile(filePath string) {
+	f, err := os.Open(filePath)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		log.Log.Reason(err).Errorf("failed to open file %s", filePath)
+	}
+	defer func() {
+		if err := f.Close(); err != nil {
+			log.Log.Reason(err).Errorf("failed to close file: %s", filePath)
+		}
+	}()
+
+	log.Log.Infof("dump log file: %s", filePath)
+	const bufferSize = 1024
+	const maxBufferSize = 512 * bufferSize
+	scanner := bufio.NewScanner(f)
+	scanner.Buffer(make([]byte, bufferSize), maxBufferSize)
+	for scanner.Scan() {
```

Comment:
Are we sure this will not block forever? If in some scenario it will, we have a big problem on our hand.

Reply:
I tested it and it wont block forever, it acts as `cat`, read until EOF and not block forever.